### ISSUE TITLE
Send epub to ebook reader

### ISSUE THREAD

**Issue by kalko1234** [janeczku/calibre-web#1054]
Please add the option to send epub files to other e-readers via e-mail (e.g. Pocktebook, InkBook)

**Comment by RafalHo** [comment]
I am also in favor of this solution.
Of course, I can use the "Send to Kindle" option, but then mobi files are sent by email, but epub files would be better.

Is there a chance to add such an option?
Alternatively, if you could add a choice, what format is to be sent?

**Comment by aakar** [comment]
FYI Amazon is going to be deprecating mobi files from being able to use the send to kindle feature.  

https://www.amazon.com/gp/help/customer/display.html?nodeId=G5WYD9SAF7PGXRNA

**Comment by DefiCzech** [comment]
Yes there should be (would be nice to have) option to send epub, mobi or any other format to my ebook email address (PocketBook in my situation).

**Comment by OzzieIsaacs** [comment]
Please update to the newest nightly version, meanwhile it's possible to send a epub to an e-reader via email

**Comment by DefiCzech** [comment]
@OzzieIsaacs yes yes amazing it works, thank you very much.

**Comment by stevetz** [comment]
@OzzieIsaacs I pulled the nightly and sending to kindle has worked for some epubs, however not all of them. I get the "There was a problem with the document(s) that you sent to Kindle" reply from amazon for a small number of books, as epub.

in calibre, i convert all files to epub (default file settings) when they're added to library - so I'd have thought that all book sshould be in same format/version of epub, I have tried with and without the calibre-pulled metadata. Happy to investigate if theres something I should be looking for?
